src/main/app/common/calculateInterest.ts

```diff
@@ -4,7 +4,7 @@ import { MomentFactory } from 'common/momentFactory'
 
 const STANDARD_RATE = 8.0
 
-function calculateRate (interest: Interest): number {
+function rate (interest: Interest): number {
```

Comment:
names of methods should be verbs. Maybe the previous name was not correct, but this one is also weird. 

ACTION REQUIRED
Please rename

Reply:
Done